tasks/verify-changed-directories.yml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: verify-changed-directories
+spec:
+  params:
+    - name: operator_name
+    - name: operator_version
+    - name: git_pr_branch
+  steps:
+    - name: get-cert-project-id
+      image: registry.access.redhat.com/ubi8-minimal
+      script: |
+        #! /usr/bin/env bash
+        # curl https://api.github.com/repos/operator-framework/community-operators/compare/master...$(params.git_pr_branch)
```

Comment:
If I remember correctly GitHub APIs are very limited and after few queries, it stops responding due to free limits. I guess there should be a way how to get a diff using just `git` cli

Reply:
"For API requests using Basic Authentication or OAuth, you can make up to 5,000 requests per hour." 
https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting
I'm open to discussing if we should be careful with that